setup.cfg

```diff
@@ -0,0 +1,37 @@
+[metadata]
+name = git-machete
+summary = Tool to rebase local branches with theirs parent branches
+description-file = README.md
+license = MIT License
+classifiers =
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 3
+    Programming Language :: Python
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+keywords = git
+author = VirtusLab
+author-email = info@virtuslab.com
```

Comment:
Pls put my personal email here, just in case... IYKWIM :) `pawel.p.lipski@gmail.com`

Reply:
OK!